packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts

```diff
@@ -75,7 +75,9 @@ export function humanFileSize(bytes, si = false, dp = 1) {
 export const checkMaxSize = (maxSize: number, file: File): string | null => {
   if (!maxSize) return null;
   if (file.size > maxSize) {
-    return translate('Above max ') + humanFileSize(maxSize, true);
+    return (
+      translate('Above file size limit of ') + humanFileSize(maxSize, true)
```

Comment:
Starting with "Above" still reads as confusing to me, who about this?
```suggestion
      translate('File size must be below ') + humanFileSize(maxSize, true)
```

Reply:
I like it, I'll change it

Reply:
[e1371c8](https://github.com/aws-amplify/amplify-ui/pull/3284/commits/e1371c81b8cc46bdb5e69a5ad606804e8949c544)